scripts/wiki-graph.py

```
@@ -0,0 +1,561 @@
+
+def load_vis_network(here: Path) -> str:
+    """vis-network-Bibliothek inline laden. Datei wird beim ersten Lauf
+    von unpkg.com nach scripts/lib/vis-network.min.js heruntergeladen
+    und danach offline verwendet."""
+    lib_path = here / "scripts" / "lib" / "vis-network.min.js"
+    if not lib_path.is_file():
+        import urllib.request
+        url = "https://unpkg.com/vis-network@9.1.9/standalone/umd/vis-network.min.js"
+        print(f"Lade vis-network herunter: {url}")
+        lib_path.parent.mkdir(parents=True, exist_ok=True)
+        urllib.request.urlretrieve(url, lib_path)
```

Ruff (0.15.12)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)